generators/entity/templates/src/main/java/package/web/rest/mapper/_EntityMapper.java

```diff
@@ -40,7 +40,8 @@
 
     List<<%= entityClass %>> <%= entityInstance %>DTOsTo<%= entityClassPlural %>(List<<%= entityClass %>DTO> <%= entityInstance %>DTOs);<%
 
-var existingMappings = [];
+// the user mapping is imported in the @Mapper annotation
+var existingMappings = ['user'];
```

Comment:
This line prevents userFromId from being generated. I felt like it was cleaner than adding another switch